packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -191,14 +197,42 @@ export class DataStores implements IDisposable {
                 }),
             pkg);
 
-        // Resolve pending gets and store off any new ones
         this.contexts.addBoundOrRemoted(remotedFluidDataStoreContext);
 
         // Equivalent of nextTick() - Prefetch once all current ops have completed
         // eslint-disable-next-line @typescript-eslint/no-floating-promises
         Promise.resolve().then(async () => remotedFluidDataStoreContext.realize());
     }
 
+    public processAliasMessage(message: ISequencedDocumentMessage): IDataStoreAliasMapping | undefined {
+        const aliasMessage = message.contents as IDataStoreAliasMessage;
```

Comment:
can we use a type guard here the validate the message is of the type we expect?

Reply:
Good point

Reply:
Adding a guard and a test for this..